ubuntuStation/forms.py

```diff
@@ -77,3 +89,11 @@ class LoginForm(FlaskForm):
     username=TextField('username', description=u'Username', validators=[DataRequired()])
     password=PasswordField('password', description=u'Password', validators=[DataRequired()])
 
+class UserForm(FlaskForm):
```

Comment:
Wasn't this added on https://github.com/erdl/survey_admin/pull/34/files#diff-c56dbff25916fe369f8214c588886a95 ?

I suspect this will lead to a merge conflict. Also will need to be re-named to `CreateUser` if will stay here ,as requested on the other PR, for code readability.